app/services/remotes/remote_case_type.rb

```diff
@@ -0,0 +1,38 @@
+module Remotes
+  class RemoteCaseType < RemoteObject
+
+    @@case_types = nil
+
+    def self.all
+      @@case_types ||= get_case_types
+    end
+
+    def self.lgfs
+      all.select { | ct| ct.roles.include?('lgfs') }
```

Comment:
A small performance improvement could be gained by using a private class instance variable here. E.g. `@lgfs ||= ...`